VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/individual-participant-status-list/individual-participant-status-list.component.ts

```diff
@@ -75,7 +76,11 @@ export class IndividualParticipantStatusListComponent extends WRParticipantStatu
             return false;
         }
 
-        if (participant.username.toLocaleLowerCase().trim() === this.adalService.userInfo.userName.toLocaleLowerCase().trim()) {
+        if (!this.loggedInUser) {
```

Comment:
setCurrentParticipant called from init, would this ever happen?

Reply:
Removed, should not happen